package.json

```diff
@@ -73,7 +73,7 @@
     "module-alias": "^2.2.2",
     "moment-timezone": "^0.5.35",
     "morgan": "~1.10.0",
-    "neverthrow": "^4.3.1",
+    "neverthrow": "^6.1.0",
```

Comment:
wait the changes described in the pr, they are not breaking changes right? Ie the semantics of the code doesnt change with or without the declaration since now neverthrow does the inference?

Reply:
Breaking changes because we no longer can import `combine` directly from `neverthrow`, so it needed to be updated when the library is updated.

Reply:
@dcshzj opps could i trouble you to rebase then + fix breaking changes then? i used combine in the latest pr that i merged

Reply:
Updated in a594951d10e2bec46691890d820d8362f901e1b6!